RESTSystem/Test/CStest.py

```diff
@@ -0,0 +1,38 @@
+import requests
```

Comment:
The content of this file is a bit too "personal". Please add at least some explanations on what it's for.